app/models/blocked_referrer.rb

```diff
@@ -1,2 +1,17 @@
 class BlockedReferrer < ActiveRecord::Base
+  # Uses the URL's host, unless it's a sites.google.com link, in which case we
+  # include the site name: "sites.google.com/sites/foobar".
```

Comment:
This took me a second to figure out that what we're doing is removing the trailing path info past `/sites/foobar/`. At least I think that's what we're doing. Is that right?

If so, would this be a better description?

```suggestion
  # Uses the URL's host, unless it's a sites.google.com link, in which case we
  # only include the site's "root" path:
  # sites.google.com/sites/foobar/baz/bat -> sites.google.com/sites/foobar
```

Reply:
Yes, exactly. How about this?


> Create a blocked referrer that blocks everything from this site.
> If the URL's host is `sites.google.com`, it removes all info after `/sites/foobar`:
> sites.google.com/sites/foobar/baz/bat -> sites.google.com/sites/foobar
> This ensures that we do not block every single Google Site URL.
> Otherwise, it simply uses the site's host.

Reply:
Ship it